service/src/main/java/greencity/mapping/tariff/GetTariffInfoForEmployeeDtoMapper.java

```diff
@@ -21,23 +20,23 @@ protected GetTariffInfoForEmployeeDto convert(TariffsInfo source) {
                 .nameUk(tariffLocation.getLocation().getRegion().getUkrName())
                 .nameEn(tariffLocation.getLocation().getRegion().getEnName())
                 .build())
-            .toList().getFirst();
+            .findFirst().orElse(null);
```

Comment:
Here`s probably better to use orElseThrow() instead of keep storing null in the application

Reply:
If I throw the exception here the method for getting all employees will not work because 
 the employee may not have info about the region.